src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css

```
@@ -0,0 +1,159 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-gray-300);
+  grid-column: 1 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 3;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: auto 590px;
```

I think you probably don't want to set the height of the second row to a fixed height of 590px, since this means that if you have more text, it'll spill outside of the component. If you want to make sure it's no shorter than 590px, you could use the [`minmax()`](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax) function.

```suggestion
  grid-template-rows: auto minmax(590px, auto);
```